Build.proj

```diff
@@ -9,6 +9,7 @@
 
   <Import Project="$(RepositoryEngineeringDir)SubsetValidation.targets" />
   <Import Project="$(RepositoryEngineeringDir)restore\optimizationData.targets" Condition="'$(DotNetBuildFromSource)' != 'true'" />
+  <Import Project="$(RepositoryEngineeringDir)targetingpacks.targets" />
```

Comment:
That's the wrong place to import this file. This import would need to go to the projects that need the crossgen2 pack. Which managed projects will use the live crossgen2 pack with this change?

Reply:
Revert

Reply:
I thought I had removed this since your first comment. My bad, will remove it in the next iteration.